react-forms/react-forms/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+    const form = event.currentTarget as HTMLFormElement;
+    const strangeFormData = new FormData(form);
+    const { username, password } = Object.fromEntries(strangeFormData);
+    console.log(`{ ${username}, ${password} }`);
```

Comment:
Same as above, though in this case it's even easier since you can just log the (non-destructured) return value of `fromEntries`.